assets/js/components/KeyMetrics/MetricsSelectionPanel/Header.js

```diff
@@ -60,20 +58,16 @@ export default function Header( { closePanel } ) {
 
 	return (
 		<SelectionPanelHeader
-			title={
-				isConversionReportingEnabled
-					? sprintf(
-							/* translators: %d: number of max allowed metrics */
-							_n(
-								'Select up to %d metric',
-								'Select up to %d metrics',
-								MAX_SELECTED_METRICS_COUNT_WITH_CONVERSION_EVENTS,
-								'google-site-kit'
-							),
-							MAX_SELECTED_METRICS_COUNT_WITH_CONVERSION_EVENTS
-					  )
-					: __( 'Select your metrics', 'google-site-kit' )
-			}
+			title={ sprintf(
```

Comment:
We need to use `__` instead of `_n` because we know that `MAX_SELECTED_METRICS_COUNT` is always greater than one.

Reply:
@eugene-manuilov this has been resolved in my latest commits.